sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -453,10 +443,19 @@ def big_data_passing_pipelinerun(name: str, pr: dict, pw: set):
     pipelinerun_name = name
     if pipelinerun_name in pw:
         pr.get('spec', {}).setdefault('workspaces', [])
+        # Change persistentVolumeClaim to volumeClaimTemplate which need Tekton version > 0.12
+        # User could modify default size of storage in the yaml file mannually if necessary.
         pr['spec']['workspaces'].append({
             "name": pipelinerun_name,
-            "persistentVolumeClaim": {
-                "claimName": pipelinerun_name
+            "volumeClaimTemplate": {
+                "spec": {
+                    "accessModes": ["ReadWriteMany"],
```

Comment:
Many of the IBM Cloud users are using block storage, can you make the default to `ReadWriteOnce`? Thanks.

Reply:
Hi @Tomcli , if `ReadWriteOnce` was set, then some cases may not work for big data pipelineruns. e.g. different tasks in the pipeline read/write the data to storage in the same time.

Reply:
Hi @fenglixa, then can we have some environment variable to change the accessModes? thanks

Reply:
OK, @Tomcli , I addressed it as your suggestion, make as an environment variable. make as resolve conversatioon now